src/util/browser.ts

```diff
@@ -341,7 +341,13 @@ export class Browser {
       return;
     }
 
-    logger.info("Run Script Started", details, contextName);
+    const isTopFrame = !frame.parentFrame();
+
+    if (isTopFrame) {
+      logger.debug("Run Script Started", details, contextName);
+    } else {
+      logger.debug("Run Script Started in iframe", details, contextName);
+    }
```

Comment:
Bit of a nit/can be cleaned up in a follow-up, but our casing for log messages is a bit all over the place. I'd suggest we move to sentence case (only capitalize first word and proper nouns, being selective about the latter)